common/placer1.cc

```diff
@@ -1021,7 +1022,7 @@ class SAPlacer
                 }
             }
 
-            if (ctx->timing_driven && int(pn->users.size()) < cfg.timingFanoutThresh) {
+            if (ctx->setting<bool>("timing_driven") && int(pn->users.size()) < cfg.timingFanoutThresh) {
```

Comment:
This, and a few other similar places, will be close to the performance critical path in the placer - I'd really like to see thing faster for checking timing_driven here

Reply:
fixed here, and added similar to placer_heap for consistency